feat(messaging): documentation for dead letter queue limits (#4195)
scaleway/docs-content: serverless/messaging/concepts.mdx

## Dead Letter Queue

> A **D**ead **L**etter **Q**ueue (DLQ), or **undelivered-message queue**, receives and holds messages that cannot be delivered to their destination queues. If you use the dead letter queue, and the storage quota is reached, messages won't be redriven to the DLQ until enough free space is available again. If you get in a situation where the quota is reached, you need to make some free space by receiving and deleting messages from any queue of your project.

Reviewer: ```suggestion
A **D**ead **L**etter **Q**ueue (DLQ), or **undelivered-message queue**, receives and holds messages that cannot be delivered to their destination queues. If you designate a queue as a DLQ and its [storage quota](/identity-and-access-management/organizations-and-projects/additional-content/organization-quotas/#queues) is reached, messages won't be redriven to the DLQ until enough free space is available again. If your DLQ is at its full quota, free up space by receiving and deleting messages from any queue in your Project.
```
